scopesim/effects/metis_lms_trace_list.py

```diff
@@ -561,8 +562,11 @@ class MetisLMSEfficiency(TERCurve):
     }
 
     def __init__(self, **kwargs):
-        self.meta = self._class_params
-        self.meta.update(kwargs)
+        # TODO: Refactor these _class_params?
+        self.meta = copy.copy(self._class_params)
+        assert "grat_spacing" in self.meta, "grat_spacing is missing from self.meta 1"
+        super().__init__(**kwargs)
+        assert "grat_spacing" in self.meta, "grat_spacing is missing from self.meta 2"
```

Comment:
I assume these assert statements are fine for now, and because this whole class is METIS specific, and nobody is going to screw too much with the current status quo in the METIS IRDB, these will not affect anything down the line. Or?